src/components/index.tsx

```diff
@@ -14,7 +20,7 @@ export { HeroSection } from "./sections/Hero.section";
 export { FAQSection } from "./sections/FAQ.section";
 
 /*
- * @component Accordioin 
+ * @component Accordioin
```

Comment:
```suggestion
 * @component Accordion
```

Didn't catch this typo before, I think we can fix it up in this PR.

Reply:
Will fix :)